README.md

```diff
@@ -109,7 +109,14 @@ To create a test development DB, for local development (e.g. virtualenv steps be
 
         npm run dev
 
-### Scheduled Events
+### Freeze/Unfreeze and Save Queryset features
+Make sure the following schemas are created;
+
+- `archives` i.e `CREATE SCHEMA archives;`
+- `dataset` i.e `CREATE SCHEMA dataset;`
+
+### Django-crontab setup
+This is used to run cron jobs that run automated scripts that are added from the UI. You need to make sure cron is installed on the web docker container. If not, run `apt-get install cron`. This has been added to the Dockerfile so it should install this automatically for new deployments (container rebuilds)
```

Comment:
I hope this doesn't interfere with native crons, specifically those that run the scheduled events ... we removed this cos it resulted in context errors

Reply:
This has been removed. Rebasing issue.